apps/backend/src/mcp/server.py

```
@@ -57,6 +57,35 @@
     infrastructure_health_check,
     troubleshoot_system_issue,
 )
+from apps.backend.src.mcp.prompts.container_workflows import (
+    deploy_application_stack,
+    troubleshoot_container_issues,
+    container_stack_maintenance,
+    scale_container_services,
+)
```

Ruff (0.12.2)

Module level import not at top of file

(E402)